diracx-client/pyproject.toml

```diff
@@ -19,21 +19,26 @@ dynamic = ["version"]
 testing = ["diracx-testing"]
 types = []
 
-[tool.setuptools.packages.find]
-where = ["src"]
-
-[build-system]
-requires = ["setuptools>=61", "wheel", "setuptools_scm>=8"]
-build-backend = "setuptools.build_meta"
-
 [project.entry-points."diracx"]
 client_class = "diracx.client.generated._client:Dirac"
 aio_client_class = "diracx.client.generated.aio._client:Dirac"
 
+[build-system]
```

Comment:
Maybe comment here that this is because of the pth file

Reply:
I was going to open an issue to move everything to using `hatchling`, it was less clear two years ago but it's now faster and objectively better than setuptools.